src/arch/host/lib/ams.c

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
commit message typos

one-way
by the main core

Reply:
ok, fixed

Reply:
still have 'mian core'